types/orderedset_test.go

```diff
@@ -1,8 +1,9 @@
 package types
 
 import (
-	"github.com/stretchr/testify/assert"
 	"testing"
+
+	"github.com/stretchr/testify/assert"
 )
```

Comment:
Plz remove the unnecessary format changes.

Reply:
It was formatted running your script `build.sh format`. I guess something changed in the Go toolchain and this is a new rule.

Should I discard it anyway?

Reply:
> It was formatted running your script `build.sh format`. I guess something changed in the Go toolchain and this is a new rule.
> 
> Should I discard it anyway?

Yep. plz discard it , I gonna check the `build.sh format` later. thx for the inconvenient. 😅

Reply:
done